hack/verify-golint.sh

```diff
@@ -22,19 +22,13 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 cd "${KUBE_ROOT}"
 
-GOLINT=${GOLINT:-"golint"}
-PACKAGES=($(go list ./internal/... | grep -v /vendor/))
-bad_files=()
-for package in "${PACKAGES[@]}"; do
-  out=$("${GOLINT}" -min_confidence=0.9 "${package}" | grep -v -E '(should not use dot imports|internal/file/bindata.go)' || :)
-  if [[ -n "${out}" ]]; then
-    bad_files+=("${out}")
-  fi
-done
-if [[ "${#bad_files[@]}" -ne 0 ]]; then
-  echo "!!! '$GOLINT' problems: "
-  echo "${bad_files[@]}"
-  exit 1
+LINT=${LINT:-golangci-lint}
+
+if [[ -z "$(command -v ${LINT})" ]]; then
+  echo "${LINT} is missing. Installing it now."
+  # See: https://golangci-lint.run/usage/install/#local-installation
+  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.53.3
+  LINT=$(go env GOPATH)/bin/golangci-lint
 fi
 
-# ex: ts=2 sw=2 et filetype=sh
+${LINT} run
```

Comment:
nit: new line

Reply:
Added

Reply:
Thanks, I added a blank line to the end of each file I created